main.go

```diff
@@ -63,13 +62,7 @@ func handleJobs(jobsC <-chan context.Job) {
 func executeTasks(tasks []*context.Task, watcher *context.Watcher) {
 	for _, task := range tasks {
 		command := task.Command
-		tokens := strings.Split(command, " ")
-		name := tokens[0]
-		var cmdArg []string
-		if len(tokens) > 1 {
-			cmdArg = tokens[1:]
-		}
-		cmd := exec.Command(name, cmdArg...)
+		cmd := exec.Command("/bin/bash", "-c", command)
```

Comment:
its not just env variables, it could be another command as well, which could have a env variable inside.... i am pretty weary of using regex for that.... too many cases to check and it could get really complicated real fast.

I am personally running sub commands inside $() which caused the issue in the first place

I was thinking of maybe 
1. detecting the os/shell and switching how the command is run. 
2. have an attribute in the goat yml/json file to indicate the os/shell.

I really dont want to over complicate stuff though